frontend/src/components/dashboard/varianceHomePage/VarianceHomePage.js

```diff
@@ -145,24 +153,12 @@ export class CustomHomePage extends Component {
   }
 
   renderDataFromURL = (params) => {
-    const {
-      issue_date_min,
-      issue_date_max,
-      expiry_date_max,
-      expiry_date_min,
-      region,
-      compliance_code,
-      major,
-      search,
-      ...remainingParams
-    } = queryString.parse(params);
+    const { region, compliance_code, major, search, ...remainingParams } = queryString.parse(
+      params
+    );
```

Comment:
This formatting is weird. `parse(params);` looks like it should all be on one line

Reply:
auto formatted by vs code

Reply:
Okay. Well, it doesn't look like Prettier formatting 🤷‍♂I wonder if this is related to the dependency update...

Getting any errors like this? https://github.com/prettier/prettier-vscode/issues/870